DocumentedExamples/Meridional_heat_transport.ipynb

```diff
@@ -5,15 +5,19 @@
    "metadata": {},
```

Comment:
<p>A comment here would be useful to describe how these observational estimates are derived. As far as I know, they are derived effectively using method 2 (but possibly with some tendency from OHC observations taken into account)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/285/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>